chooseBuild.hbs

```diff
@@ -0,0 +1,18 @@
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
That tag [might be dropped](https://stackoverflow.com/a/6771584/2008562) if we support only latest IE/Edge versions.